packages/e2e/plugins/fsPlugin/index.ts

```diff
@@ -0,0 +1,22 @@
+import { rmdir } from "fs";
+
+const fsPlugin = (on: Cypress.PluginEvents): void => {
+  on("task", {
+    deleteFolder(folderName) {
+      console.log("deleting folder %s", folderName);
+
+      return new Promise((resolve, reject) => {
+        // eslint-disable-next-line consistent-return
+        rmdir(folderName, { maxRetries: 10, recursive: true }, (err) => {
+          if (err && err.code !== "ENOENT") {
+            console.error(err);
+            return reject(err);
+          }
```

Comment:
This feels a bit wrong...What happens is that if (and only if) err.code is `ENOENT` you console it and `reject()`, which (and correct me if I'm wrong) is the promise equivalent of throwing errors. There are two points:
1. I feel it would be ok to not fail if the folder doesn't exist (and just `console.log` it, so it's not as silent)
2. If there's another error, I believe you would like it to fail (by throwing an exception) rather than failing silently an messing up the tests (for no apparent reason)

Reply:
Btw. :+1: :+1: on using the separate plugin file

Reply:
but what happens here is that it only `rejects` and `console.error` if the error code is anything *but* `ENOENT`

Reply:
Right…sry I was tired lol